experimenter/manifesttool/releases.py

```diff
@@ -60,6 +60,9 @@ def discover_tagged_releases(
 
     min_version = Version(max_major_version - 4, 0, 0)
 
+    if strategy.minimum_version is not None:
+        min_version = max(min_version, strategy.minimum_version)
+
```

Comment:
Could you do
```suggestion
    if strategy.minimum_version:
        min_version = max(min_version, strategy.minimum_version)

```

Reply:
Yes, but I prefer the explicit None check, which makes it clear its either something or None. Also

> Explicit is better than implicit.
according to the zen of python (https://peps.python.org/pep-0020/) :P

Reply:
Oh yeah explicitly checking for `None` has semantic meaning, good to keep here 👍